### PR TITLE
Adds dap

### PR DESCRIPTION
**Pull request by sadiejay**

### CODE REVIEW

File: _includes/scripts.html [hunk 1]
```diff
@@ -1,2 +1,3 @@
 <script src="{{ site.baseurl }}/assets/js/vendor/uswds.min.js"></script>
-<script src="{{ site.baseurl }}/assets/js/styleguide.js"></script>
\ No newline at end of file
+<script src="{{ site.baseurl }}/assets/js/styleguide.js"></script>
+<script async id="_fed_an_ua_tag" src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js?agency=DOC&amp;subagency=CEN" type="text/javascript"></script>
```
Comment (curt-mitch-census) [inline _includes/scripts.html:3]: I think this should be moved to `head.html` so that it's loaded within the `head` HTML tags.

**Review by sadiejay — COMMENTED** [review]
Thank you for catching this! Will move these to head!

File: _includes/scripts.html (hunk 1 above)
Comment (sadiejay) [reply _includes/scripts.html]: @curt-mitch-census I think I fixed this issue and would appreciate input!

Comment (curt-mitch-census) [reply _includes/scripts.html]: @sadiejay Sorry, I should have been more specific: you were right to move the dap script to the `head.html`, according to their [suggested documentation](https://github.com/digital-analytics-program/gov-wide-code/blob/master/documentation/GSA%20DAP%204.1%20-%20Quick%20Guide.pdf). However, I think you can keep the `styleguide.js` file within the `scripts.html` file along with the `uswds.min.js` file to keep the non-analytics JavaScript files in one place. My apologies for the mixup!

File: _includes/head.html [hunk 2]
```diff
@@ -59,3 +59,6 @@
     <link rel="alternate" type="text/html" href="{{ site.baseurl }}/assets/css/styles.css">
   {% endif %}
 {% endif %}
+
+<!-- style guide -->
+<script src="{{ site.baseurl }}/assets/js/styleguide.js"></script>
```
Comment (curt-mitch-census) [inline _includes/head.html:64]: It looks like you can remove this script reference here and place it back in the `_includes/scripts.html` file.

File: _includes/scripts.html [hunk 3]
```diff
@@ -1,2 +1 @@
 <script src="{{ site.baseurl }}/assets/js/vendor/uswds.min.js"></script>
-<script src="{{ site.baseurl }}/assets/js/styleguide.js"></script>
```
Comment (curt-mitch-census) [inline _includes/scripts.html:2]: Same comment as above in the `_includes/head.html` file, but you can restore this reference to `styleguide.js` here and remove the script tags for it in the `_includes/head.html` file in order to keep similar JS files grouped together.

**Review by curt-mitch-census — APPROVED** [review]
I left some small comments on the location of the `script` tag for the `styleguide.js` file, otherwise this LGTM! 👍

File: _includes/head.html (hunk 2 above)
Comment (sadiejay) [reply _includes/head.html]: Done!

File: _includes/scripts.html (hunk 3 above)
Comment (sadiejay) [reply _includes/scripts.html:2]: And done! Thank you!

**Pull request merged**